### PR TITLE
Sign in page

### PR DESCRIPTION
**Pull request by pariyar07**

### CODE REVIEW

File: js/sign-up.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const password = document.querySelector('#password');
+const checkPassword = document.querySelector('#checkpassword');
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    checkInputs();
+})
+
+function checkInputs() {
+    const usernameValue = username.value.trim();
```
Comment (sanjitsarkar) [inline js/sign-up.js:15]: you can use required attribute on input tag to make it mandatory

File: styles/footer.css [hunk 2]
```diff
@@ -7,7 +7,13 @@
     height: 40vh;
     background-color: var(--secondary-bg);
     border-top-left-radius: 15px;
```
Comment (sanjitsarkar) [inline styles/footer.css:9]: you can use rem here

**Review by sanjitsarkar — APPROVED** [review]

File: js/sign-up.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    if(checkPasswordValue === "") {
+        setErrorFor(checkPassword, "Please enter a same password");
+    } else if(checkPasswordValue !== passwordValue) {
+        setErrorFor(checkPassword, "Password does not match");
+    } else {
+        setSuccessFor(checkPassword)
+    }
+}
```
Comment (saurabhchirde) [inline js/sign-up.js:47]: no need to validate these input fields in js, these can be validated in HTML itself.

**Review by saurabhchirde — APPROVED** [review]
The code seems clean and well structured. GTM

File: pages/sign-in.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            </li>
+            <li><a href=""><i class="far fa-user-circle"></i></a>
+                <p>Account</p>
+            </li>
+        </div>
+    </nav>
+    <!-- sign-up section -->
+    <section class="sign-up-wrapper">
+        <div class="sign-up-left">
+            <img src="../assets/images/sign-up.png" alt="">
```
Comment (Aditi0505) [inline pages/sign-in.html:52]: Suggestion: Please provide an alt description.

File: pages/sign-up.html [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+            </li>
+            <li><a href=""><i class="far fa-user-circle"></i></a>
+                <p>Account</p>
+            </li>
+        </div>
+    </nav>
+    <!-- sign-up section -->
+    <section class="sign-up-wrapper">
+        <div class="sign-up-left">
+            <img src="../assets/images/sign-up.png" alt="">
```
Comment (Aditi0505) [inline pages/sign-up.html:52]: Same as above.

**Review by Aditi0505 — APPROVED** [review]
Added one suggestion. Rest looks good to merge after changes.

**Pull request merged**